src/components/search/History/SearchHistory.tsx

```diff
@@ -5,6 +5,7 @@ import Button from 'src/components/ui/Button'
 import Icon from 'src/components/ui/Icon'
 import Link from 'src/components/ui/Link'
 import useSearchHistory from 'src/sdk/search/useSearchHistory'
+import './search-history.scss'
```

Comment:
Put this CSS import statement inside the gatsby-browser.js to prevent warnings.

Reply:
Yeah. After the changes from #434, we're now adding the SCSS imports to this `components.scss` file.

`search-history.scss`, `search-input.scss`, and `suggestions.scss` are already being imported there so I think we can remove these SCSS imports from these 3 component files.

https://github.com/vtex-sites/base.store/blob/5b1175b323a5955ede31ca90d98a701532f24f3b/src/styles/global/components.scss#L20

https://github.com/vtex-sites/base.store/blob/5b1175b323a5955ede31ca90d98a701532f24f3b/src/styles/global/components.scss#L10

https://github.com/vtex-sites/base.store/blob/5b1175b323a5955ede31ca90d98a701532f24f3b/src/styles/global/components.scss#L22